### PR TITLE
Feat/style input styles

### PR DESCRIPTION
**Pull request by cristianoliveira**
- Add input styles
 - Introduce mask for inputs

Screenshot 2022-02-03 at 23 16 27

### CODE REVIEW

File: src/UserDataForm.js [hunk 1]
```diff
@@ -43,92 +56,86 @@ class UserDataForm extends React.Component {
...
-        <label>
-          Postal Code:
-          <input
+            onChange={this.handleInputChange}
+          />
+        </SideBySide>
+        <Input
+          description="Birthday"
+          placeholder="01.01.2001 (dd.mm.yyyy)"
+          mask="99.99.9999"
```
Comment (juanibiapina) [inline src/UserDataForm.js:82]: The date should now be ISO8601. I changed in the backend already.

File: src/UserDataForm.js [hunk 2]
```diff
@@ -43,92 +56,86 @@ class UserDataForm extends React.Component {
...
+          name="birthday"
+          type="text"
```
Comment (juanibiapina) [inline src/UserDataForm.js:84]: Why not use type date here?

File: src/UserDataForm.js [hunk 3]
```diff
@@ -43,92 +56,86 @@ class UserDataForm extends React.Component {
...
-    )
+            onChange={this.handleInputChange}
+          />
+        </SideBySide>
+        <Input
+          description="Phone Number"
+          placeholder="012224053535"
+          mask="999999999999"
+          name="phone_number"
+          type="text"
```
Comment (juanibiapina) [inline src/UserDataForm.js:120]: And type tel here?

File: src/UserDataForm.js [hunk 4]
```diff
@@ -43,92 +56,86 @@ class UserDataForm extends React.Component {
 
   render() {
     return (
-      <form onSubmit={this.handleSubmit}>
-        <label>
-          Name:
-          <input
+      <StyledForm onSubmit={this.handleSubmit}>
+        <h4>Bouldering Booking form</h4>
```
Comment (juanibiapina) [inline src/UserDataForm.js:60]: How about we change this to `User Data` or something like that?

File: src/UserDataForm.js [hunk 5]
```diff
@@ -43,92 +56,86 @@ class UserDataForm extends React.Component {
...
+        <Input
+          description="USC Membership No"
+          placeholder="111111111"
+          mask="999999999"
+          name="usc_number"
+          type="text"
+          value={this.state.user.usc_number}
+          onChange={this.handleInputChange}
+        />
+        <SubmitButton type="submit" value="Submit" />
```
Comment (juanibiapina) [inline src/UserDataForm.js:133]: How about we change this submit button text to `Save User Data` or something like that? It makes it more clear that we're just saving the user data for now.

**Comment by cristianoliveira** [comment]
I forgot to include email 😅 
I'll take the chance and address the suggestion

File: src/UserDataForm.js (hunk 4 above)
Comment (cristianoliveira) [reply src/UserDataForm.js:60]: I think something related to user data better represents what the form does. Maybe "personal data"

File: src/UserDataForm.js (hunk 5 above)
Comment (cristianoliveira) [reply src/UserDataForm.js:141]: Good one, I'll include a disclaimer about the data being stored in the browser for later consumption

**Pull request merged**